### ISSUE TITLE
[BUG] SyntaxError: invalid decimal literal in OpenAPI APP

### ISSUE THREAD

**Issue by azgaviperr** [Shuffle/Shuffle#932]
**Describe the bug**
Using a shared OpenAPI app from the market place I got an issue with a node not able to execute. 

OpenAPI : Wallix_Bastion_API_v3.3_LTS

logs of the worker 

``` 
docker logs -f 63d4647a0833ae86c9ef717be484b76fd5a7b4a5d2769b0c40501f2d8fa8a523
2022/09/09 11:49:11 [ERROR] Failed setting up Opensearch: EOF
2022/09/09 11:49:11 [ERROR] Failed to run worker init: EOF
2022/09/09 11:49:11 [INFO] Setting up worker environment
2022/09/09 11:49:11 [INFO] Running with timezone Europe/Amsterdam and swarm config "runn"
2022/09/09 11:49:11 [INFO] Running normal execution with auth 583928de-8831-40ff-8ce0-594f206b07e0 and ID a11e8b83-0d47-471f-b8ae-da40cc3fbd65
2022/09/09 11:49:11 [DEBUG] Looking for subflow in []string{"a2dcc39c-82cf-4d15-8227-8b5c661f0774", "d2243bab-d4a7-450c-b573-d138aaeb8986", "5beccf27-61c9-429d-b7fc-eacbec1ccc76", "af3337e3-33ca-4588-909c-38acb7296001", "377b2f16-1048-410b-8bdd-8ed4b4995a00", "32e9d244-4cec-4fb6-b279-9359c22a0533", "7cf13ff0-54a1-4f3e-8582-8fbfb22ef796"} to check execution pattern as child of a2dcc39c-82cf-4d15-8227-8b5c661f0774
2022/09/09 11:49:11 

Environments: [Shuffle]. Source: 54ef000c-8047-46ce-801a-d57439be3244. 1 env = webserver, 0 or >1 = default. Subflow exists: false

2022/09/09 11:49:11 

[DEBUG] Running OPTIMIZED execution (not manual)

2022/09/09 11:49:11 [DEBUG] OLD HOSTNAME: http://shuffle-backend:5001
2022/09/09 11:49:11 

Starting webserver on port 40623 with hostname: 172.22.0.4

2022/09/09 11:49:11 NEW HOSTNAME: http://172.22.0.4:40623
2022/09/09 11:49:11 [INFO][a11e8b83-0d47-471f-b8ae-da40cc3fbd65] STARTACTION: a2dcc39c-82cf-4d15-8227-8b5c661f0774
2022/09/09 11:49:11 [INFO][a11e8b83-0d47-471f-b8ae-da40cc3fbd65] shuffle.Actions: 7 + Special shuffle.Triggers: 0
2022/09/09 11:49:11 [DEBUG][a11e8b83-0d47-471f-b8ae-da40cc3fbd65] Getting info for a11e8b83-0d47-471f-b8ae-da40cc3fbd65. Extra: 0
2022/09/09 11:49:11 [DEBUG][a11e8b83-0d47-471f-b8ae-da40cc3fbd65] Inside execution results with 1 / 7 results
2022/09/09 11:49:11 [DEBUG][a11e8b83-0d47-471f-b8ae-da40cc3fbd65] get_target_password:d2243bab-d4a7-450c-b573-d138aaeb8986 has no status result yet. Should execute.
2022/09/09 11:49:11 [DEBUG][a11e8b83-0d47-471f-b8ae-da40cc3fbd65] Docker Container Status: exited, should kill: Wallix_Bastion_API_v3-3_LTS_1-0-0_d2243bab-d4a7-450c-b573-d138aaeb8986_a11e8b83-0d47-471f-b8ae-da40cc3fbd65
2022/09/09 11:49:11 [INFO][a11e8b83-0d47-471f-b8ae-da40cc3fbd65] Time to execute d2243bab-d4a7-450c-b573-d138aaeb8986 (Checkout) with app Wallix_Bastion_API_v3.3_LTS:1.0.0, function get_target_password, env Shuffle with 8 parameters.
2022/09/09 11:49:11 [DEBUG] Adding a11e8b83-0d47-471f-b8ae-da40cc3fbd65_d2243bab-d4a7-450c-b573-d138aaeb8986 to cache. Name: get_target_password
2022/09/09 11:49:11 [DEBUG] Running app get_target_password in docker NORMALLY as there is no delay set with identifier Wallix_Bastion_API_v3-3_LTS_1-0-0_d2243bab-d4a7-450c-b573-d138aaeb8986_a11e8b83-0d47-471f-b8ae-da40cc3fbd65
2022/09/09 11:49:11 [DEBUG] 2 - Identifier: Wallix_Bastion_API_v3-3_LTS_1-0-0_d2243bab-d4a7-450c-b573-d138aaeb8986_a11e8b83-0d47-471f-b8ae-da40cc3fbd65-b8932a3e-f14e-4bef-a318-33cbe00448a0
2022/09/09 11:49:12 [DEBUG] Container 8fd92b1196112241063c480aef056b9be3feb5ad06f13389ea5e646a09a5dfa3 was created for Wallix_Bastion_API_v3-3_LTS_1-0-0_d2243bab-d4a7-450c-b573-d138aaeb8986_a11e8b83-0d47-471f-b8ae-da40cc3fbd65-b8932a3e-f14e-4bef-a318-33cbe00448a0
2022/09/09 11:49:12 [INFO] Handling NON-default execution source 54ef000c-8047-46ce-801a-d57439be3244 - NOT waiting or validating!
2022/09/09 11:49:12 [DEBUG] Deployed container ID 8fd92b1196112241063c480aef056b9be3feb5ad06f13389ea5e646a09a5dfa3
2022/09/09 11:49:12 [DEBUG] Normal deploy ret: %!s(<nil>)
2022/09/09 11:49:12 [DEBUG][a11e8b83-0d47-471f-b8ae-da40cc3fbd65] repeat_back_to_me:7cf13ff0-54a1-4f3e-8582-8fbfb22ef796 has no status result yet. Should execute.
2022/09/09 11:49:12 [DEBUG][a11e8b83-0d47-471f-b8ae-da40cc3fbd65] Docker Container Status: exited, should kill: Shuffle-Tools_1-2-0_7cf13ff0-54a1-4f3e-8582-8fbfb22ef796_a11e8b83-0d47-471f-b8ae-da40cc3fbd65
2022/09/09 11:49:12 [INFO][a11e8b83-0d47-471f-b8ae-da40cc3fbd65] Time to execute 7cf13ff0-54a1-4f3e-8582-8fbfb22ef796 (Service) with app Shuffle Tools:1.2.0, function repeat_back_to_me, env Shuffle with 1 parameters.
2022/09/09 11:49:12 [DEBUG] Adding a11e8b83-0d47-471f-b8ae-da40cc3fbd65_7cf13ff0-54a1-4f3e-8582-8fbfb22ef796 to cache. Name: repeat_back_to_me
2022/09/09 11:49:12 [DEBUG] Running app repeat_back_to_me in docker NORMALLY as there is no delay set with identifier Shuffle-Tools_1-2-0_7cf13ff0-54a1-4f3e-8582-8fbfb22ef796_a11e8b83-0d47-471f-b8ae-da40cc3fbd65
2022/09/09 11:49:12 [DEBUG] 2 - Identifier: Shuffle-Tools_1-2-0_7cf13ff0-54a1-4f3e-8582-8fbfb22ef796_a11e8b83-0d47-471f-b8ae-da40cc3fbd65-1c5b7962-3949-4e2f-acd7-94339d72b57b
2022/09/09 11:49:12 [DEBUG] Container e5955759d96d5d74468f831c793414830c4234aca714511d9a51c1fed5008c7d was created for Shuffle-Tools_1-2-0_7cf13ff0-54a1-4f3e-8582-8fbfb22ef796_a11e8b83-0d47-471f-b8ae-da40cc3fbd65-1c5b7962-3949-4e2f-acd7-94339d72b57b
2022/09/09 11:49:12 [INFO] Handling NON-default execution source 54ef000c-8047-46ce-801a-d57439be3244 - NOT waiting or validating!
2022/09/09 11:49:12 [DEBUG] Deployed container ID e5955759d96d5d74468f831c793414830c4234aca714511d9a51c1fed5008c7d
2022/09/09 11:49:12 [DEBUG] Normal deploy ret: %!s(<nil>)
2022/09/09 11:49:13 [DEBUG] In get stream results with body length 113
2022/09/09 11:49:13 [INFO][a11e8b83-0d47-471f-b8ae-da40cc3fbd65] Got result SUCCESS from Service with app Shuffle Tools:1.2.0
2022/09/09 11:49:13 [INFO] Setting value of Service (7cf13ff0-54a1-4f3e-8582-8fbfb22ef796) in workflow a11e8b83-0d47-471f-b8ae-da40cc3fbd65 to SUCCESS (1)
2022/09/09 11:49:13 [WARNING] Error: EOF
2022/09/09 11:49:13 

[WARNING] Failed to get workflow in fix it up: EOF

2022/09/09 11:49:13 [DEBUG][a11e8b83-0d47-471f-b8ae-da40cc3fbd65] Running setexec with status EXECUTING and %!s(int=2) results
2022/09/09 11:49:13 [DEBUG][a11e8b83-0d47-471f-b8ae-da40cc3fbd65] Getting info for a11e8b83-0d47-471f-b8ae-da40cc3fbd65. Extra: 0
2022/09/09 11:49:13 [DEBUG][a11e8b83-0d47-471f-b8ae-da40cc3fbd65] Inside execution results with 2 / 7 results
2022/09/09 11:49:15 [INFO][a11e8b83-0d47-471f-b8ae-da40cc3fbd65] VALIDATION. Status: EXECUTING, shuffle.Actions: 7, Extra: 0, Results: 2. Parent: "47255cd9-216d-4132-95ce-24631158daf1"
2022/09/09 11:49:15 [DEBUG][a11e8b83-0d47-471f-b8ae-da40cc3fbd65] Should send full result to http://shuffle-backend:5001
2022/09/09 11:49:15 [DEBUG][a11e8b83-0d47-471f-b8ae-da40cc3fbd65] NEWRESP (from backend): {"success": true, "reason": "success"}
``` 
logs from the container created for the wallix app 
``` 
docker logs 8fd92b1196112241063c480aef056b9be3feb5ad06f13389ea5e646a09a5dfa3
  File "/app/app.py", line 8
    class Wallix_Bastion_API_v3.3_LTScc2cc2d04585436f9a9cf2874ca4206b(AppBase):
                                 ^
SyntaxError: invalid decimal literal
```

**Comment by azgaviperr** [comment]
Tried to rebuild the app using app_sdk 1.0.4 and 1.0.5 same issue

**Comment by azgaviperr** [comment]
Downloaded the yaml of the app and renamed it "Wallix Bastion API FIX" node is now working. 

Tested to import the broken app from cloud and importing the yam same issue on 3 Shuffle instance. 
Tested the import of the FIX version on 2 instance and working on both

**Comment by frikky** [comment]
A quick look at it looks like there's indeed a dot in the name that's causing the issue. This has so far only been fixed on the frontend which is probably why it's broken like that.

**Comment by frikky** [comment]
Update: Fixed in next releases by sending app names themselves through the same checks as functions.

**Comment by frikky** [comment]
Should be fixed in 1.1! Closing
